### PR TITLE
Feature/yilin2

### PR DESCRIPTION
**Pull request by lvyl9909**
some changes to tutorial pages

### CODE REVIEW

**Review by ylimezhang — CHANGES_REQUESTED** [review]
Great! could you add margin and difficulty in the "Assertions for IRV winners" part?

**Review by lvyl9909 — COMMENTED** [review]
added margin as a new page

File: app/tutorial/components/margin-content.tsx [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+      {/* Navigation Links */}
+      <div className="flex justify-between items-center mt-12">
+        {/* Back Button - Left aligned */}
+        <div>
+          <Link
+            href="/tutorial/usingassertion"
+            className="text-black hover:text-gray-500 flex items-center"
+          >
+            <span className="mr-2 text-black">←</span> Back
```
Comment (Mirrorgo) [inline app/tutorial/components/margin-content.tsx:99]: do not use "←", please use an icon

File: app/tutorial/usingassertion/page.tsx [hunk 2]
```diff
@@ -22,14 +30,27 @@ const UsingAssertionsPage: React.FC = () => {
...
+          style={{
+            marginLeft: collapsed ? 0 : sidebarWidth, // 根据侧边栏状态动态调整位置
+            transition: "margin-left 0.3s ease",
+          }}
+        >
+          {/* 面包屑 */}
+          <Breadcrumbs paths={breadcrumbPaths} />
+        </div>
+
+        {/* 内容部分 */}
```
Comment (Mirrorgo) [inline app/tutorial/usingassertion/page.tsx:45]: These snippets are used multiple times. You should use a component to reduce duplicate code

File: app/tutorial/components/margin-content.tsx (hunk 1 above)
Comment (lvyl9909) [reply app/tutorial/components/margin-content.tsx]: sry, fixed

File: app/tutorial/usingassertion/page.tsx (hunk 2 above)
Comment (lvyl9909) [reply app/tutorial/usingassertion/page.tsx:45]: added a margincontainer component for breadcrumb

**Review by Mirrorgo — APPROVED** [review]

File: app/tutorial/assertion/page.tsx [hunk 3]
```diff
@@ -21,11 +28,18 @@ const AssertionPage: React.FC = () => {
...
+        <MarginContainer collapsed={collapsed} sidebarWidth={sidebarWidth}>
+          <Breadcrumbs paths={breadcrumbPaths} />
+        </MarginContainer>
+
+        {/* Assertion content */}
         <AssertionContent sidebarWidth={sidebarWidth} collapsed={collapsed} />
 
         {/* Footer Section */}
         <p className="p-4 text-sm text-gray-500 text-center border-t mt-8">
-          By sharing your files or using our service, you agree to our{" "}
```
Comment (ylimezhang) [inline app/tutorial/assertion/page.tsx:28]: TC已经有对应组件了，用upload里面的term-and-privacy.tsx

**Review by ylimezhang — CHANGES_REQUESTED** [review]

File: app/tutorial/assertion/page.tsx (hunk 3 above)
Comment (lvyl9909) [reply app/tutorial/assertion/page.tsx:28]: fixed

**Review by ylimezhang — APPROVED** [review]
LGTM！

**Pull request merged**